src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -8,7 +8,6 @@
 
 from sentry.loader.browsersdkversion import get_browser_sdk_version
 from sentry.models import Project, ProjectKey
-from sentry.relay import config
```

Comment:
i don't see any reason to have this function and add dependency between `sentry.web.frontend` and `sentry.relay.config`. So I removed it and use directly similar t0 line 65.